doc/examples/onboarding.example.yaml

```diff
@@ -38,6 +38,12 @@ addon:
     - ampResourceLimits.requests.memory
     - ampResourceLimits.limits.cpu
     - ampResourceLimits.limits.memory
+  secretNameKeyMapping:
+    ampSecretsNameOverride:
```

Comment:
@jcabrerizo in this example, the following should be clear to the seller:

1. Secret name in the AWS Secrets Manager
2. Name of the Kubernetes secret (sometimes it must have a particular name)
3. Key in the K8S secret

There are other nuances - AWS Secret can be plaintext or can be JSON. We may require it always to be in JSON so that the key in the secret matches the key in the Secrets Manager. 

One thing that I propose:

rename `secretKeyMapping` to `secretMapping`
Fields:
```
    secretName (that is the K8S secret name) 
         - key1
         - key2
 
example:
       accesskey (k8s secret is accesskey, AWS secret is lacework-accesskey)
            - secretKey
```

The Secret name in the AWS Secrets manager will be `${addOnName}-${secretName}`  - this is needed for namespacing and we cannot accept secret name as is from the partner, since we have to use namespacing.

Reply:
Hi @shapirov103, thanks for talking a look. 
That example file contained from the beginning references to the future AMP input file, that's why it had references to the AMP needed secret.

I have updated the example removing all Cloudsoft AMP references and adding comments per each line.

About the name of the property, it's a quick change I'd could do if you really insist your name is better, but I think `secretNameKeyMapping` highlights better the fact that this map is between the secret name and the key where the value is, and never the secret itself. I'm sure no vendor will put a secret in the issue, but I'm trying to avoid them even asking if that is what we expect. But as I said, as none english/aws native, maybe my view of the property name is not the best.

Finally, about the first point in the enumeration, for confirming we are aligned, we control the AWS Secrets Manager, not the seller; they will provide us the data (json) we need to put for out test, and we will store it following something similar to `${sellerMarketPlaceAlias}/${addon.name}/${addon.version}/$secretName}`. Then, that secret name will used to add into the `${addon.namespace}` a secret with the keys listed, and the property for reference the add-on will be passed as for install the add-on. I think that covers what is needed

Reply:
Hi @jcabrerizo , 

secretMapping is mapping of the secret in AWS Secret Manager to Kubernetes secret. Mapping assumes name and keys. 
SecretNameKeyMapping - is mapping of secret name to keys, or secret to name keys and it is less clear to me.

In reality, partners submitting manifest should not even know or care that we use AWS Secrets manager. To them, it is just a secret (kubernetes secret) and we ask to describe that secret. So maybe it should named just `secrets`.  

It is subjective I agree, not insisting. Maybe let's hear opinions of other folks. If it appears confusing only to me, no worries, I will approve.

Reply:
I changed it, I trust your naming criteria more than mine. Tks for the feedback